src/Service/RevisionSaver.php

```diff
@@ -95,6 +95,7 @@ private function getEditParams( Revision $revision, EditInfo $editInfo = null )
 	 */
 	private function getEditInfoParams( $editInfo ) {
 		$params = array();
+		$assertions = array();
```

Comment:
The array can contain items, as you can see on line 106. Don't ask me what uses it - if you want I can remove the complete array.